modules/ai_engine_embedding/src/Form/AiEngineEmbeddingSettings.php

```diff
@@ -18,6 +20,13 @@ class AiEngineEmbeddingSettings extends ConfigFormBase {
    */
   const CONFIG_NAME = 'ai_engine_embedding.settings';
```

Comment:
This is not a part of your current work, but I'm just now realizing that we should have a config/install for this settings file.

Reply:
Ah that makes sense.  I can add that as part of this.

Reply:
Done!